infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -366,7 +363,7 @@ private void trimPrefix() {
         // (i.e. trim only complete segments)
         long endSegment = dataStore.getStartingAddress() / RECORDS_PER_LOG_FILE - 1;
```

Comment:
You can refactor this into a `getEndSegment` method, it seems like its being used in multiple places

Reply:
changed to use getStartingSegment. In my new GC patch #1921 , I extracted the linear addressing method: address / RECORDS_PER_LOG_FILE  and cleaned up all usages of it.